Regular-Question-Answers/merge-k-sorted-lists.cpp

```diff
@@ -0,0 +1,46 @@
+// Author: Anıl Dervişoğlu
+// Question Link: https://leetcode.com/problems/merge-k-sorted-lists/
+
+// If there were to be 2 distinct lists, we could have used two pointers method to sort it easily.
+// However, since there are k lists, we need to find the order of smallest elements in all of them in the least possible time complexity.
+// This can be done using a multiset or similar data structure in O(nlogn) complexity where n is the total number of elements in all lists.
+// The idea is similar to merge sort, where we merge two lists in sorted order.

```

Comment:
Her Linked List'i ikili ikili sıralama yöntemine göre avantaj ve dezavantajları (zaman, bellek kullanımı vs.) belirtilebilir.

Reply:
İkili ikili sıralamak biraz karmaşık olabilir ama kullanılabilir çözüm metotlarından birini daha anlatmak dediğin gibi mantıklı olur. Teşekkürler!